mu/interface.py

```diff
@@ -850,6 +850,19 @@ def enable(self, sibling):
         self.setAcceptDrops(True)
         sibling.setAcceptDrops(True)
 
+    def show_confirm_overwrite_dialog(self):
+        msg = QMessageBox()
+        msg.setIcon(QMessageBox.Information)
+        msg.setText("File already exists; overwrite it?")
+        msg.setWindowTitle("File already exists")
+        msg.setStandardButtons(QMessageBox.Ok | QMessageBox.Cancel)
```

Comment:
I'd be inclined to leave OK as default; at least for computer to device transfers, as you're probably going to want to overwrite files on the device with a new one pretty often. Going the other way might be better to have cancel as default though